docs/editor/workspaces.md

```diff
@@ -9,39 +9,91 @@ MetaDescription: Learn about Visual Studio Code workspaces
+- `.code-workspace` / untitled workspace: user, workspace, folder
+- folder: user, workspace & folder (is the same)
+
 ## Multi-root workspaces
 
-You can also have more than one root folder in a VS Code workspace through a feature called [Multi-root workspaces](/docs/editor/multi-root-workspaces.md). TBD Why you would use multi-root workspaces, briefly show UI and `.code-workspace` file. Leave details to existing doc.
+[Multi-root workspaces](/docs/editor/multi-root-workspaces.md) are an advanced capability of VS Code that allow you to configure multiple distinct folders to be part of the workspace. Instead of opening a folder as workspace, you will open a `<name>.code-workspace` JSON file that lists the folders of the workspace. For example:
+
+```json
+{
+	"folders": [
+		{
+			"path": "my-folder-a"
+		},
+		{
+			"path": "my-folder-b"
+		}
+	],
+    "settings": {
+        ...
+    }
+}
+```
+
+The presence of this `.code-workspace` file makes it even possible to include workspace specific settings (or launch / task configurations) that should apply to all folders of the workspace.
+
+### Untitled multi-root workspaces
+
+We try to make it very easy for you to add or remove folders in your workspace. Most often you start off with opening a single folder in VS Code, only then to add more folders as you see fit. The very first time you add a second folder to a workspace with only one folder, VS Code will automatically enter an "untitled" workspace. In the background we automatically maintain a `untitled.code-workspace` file for you that already contains all the folders and workspace settings from your current session. The workspace will remain "untitled" until you decide to save it anywhere on disk.
+
+![Untitled multi-folder workspace](images/workspaces/untitled-workspace.png)
+
+*An untitled multi-root workspace opened in VS Code*
+
+There is really no difference between an untitled workspace and a saved workspace other than the fact that an untitled workspace is automatically created for you for your convinience and will always restore until you save it. We automatically delete untitled workspaces (after asking you for confirmation) when you close a window where the untitled workspace is opened in.
+
+**Note:** if you constantly see many windows restoring when you start VS Code, make sure to review the ones with untitled workspaces in them. To stop restoring untitled workspaces, simply close the window and confirm the prompt to not save the untitled workspace.
```

Comment:
How common is this behavior? Would it be better to have in Common Questions if few people hit this?